src/server.js

```diff
@@ -58,6 +64,16 @@ const handleRender = (req, res, data) => {
   }
 };
 
+const handleSpaceRender = (req, res) => {
+  const id = req.path.split('/')[2];
```

Comment:
Comment or helper function to see what this is actually doing. Maybe `id` can be passed in a cleaner way.